src/resmom/mom_main.c

```diff
@@ -6143,21 +6143,18 @@ do_rpp(int stream)
 
 /**
  * @brief
- *	wrapper function for do_rpp
+ *	wrapper function for do_rpp which calls infinitely
  *
  * @param[in] fd - file descriptor
  *
  * @return Void
  *
  */
-#define MAX_RPP_LOOPS 3
 void
 rpp_request(int fd)
 {
 	int	stream;
-	int	i;
-	/* To reduce rpp process storm reducing max do_rpp processing to 3 times */
-	for (i=0 ; i < MAX_RPP_LOOPS ; i++) {
+	for (;;) {
```

Comment:
Have you removed this code temporarily? Hope you are adding back this code again.

Reply:
Added back